Description: Update links in the documentation to point to copies of PostScript documents on VIGRA homepage.
Author: Jakub Wilk <jwilk@debian.org>
Forwarded: not-needed
Last-Update: 2011-03-02

Index: b/docsrc/index.dxx
===================================================================
--- a/docsrc/index.dxx
+++ b/docsrc/index.dxx
@@ -67,7 +67,7 @@
     in: B. J&auml;hne, H. Hau&szlig;ecker, P. Gei&szlig;ler: "Handbook on
     Computer Vision and  Applications", volume 3, Academic Press, 1999.
     If you don't have the book,  you may read a
-    <a href="documents/GenericProg2D.ps">draft of this article</a>. A shorter
+    <a href="http://hci.iwr.uni-heidelberg.de/vigra/doc/vigra/documents/GenericProg2D.ps">draft of this article</a>. A shorter
     article <em><a href="http://hci.iwr.uni-heidelberg.de/people/ukoethe/papers/index.php#cite_GenericProg2DCppReport">STL Style Generic Programming with Images</a></em>
     describing some of the
     ideas has appeared in the January 2000 issue of
Index: b/include/vigra/accessor.hxx
===================================================================
--- a/include/vigra/accessor.hxx
+++ b/include/vigra/accessor.hxx
@@ -51,7 +51,7 @@
     are bound to what <TT>operator*()</TT> returns, if this method exists at
     all. Encapsulating access in an accessor enables a better
     decoupling of data structures and algorithms.
-    <a href="documents/DataAccessors.ps">This paper</a> contains
+    <a href="http://hci.iwr.uni-heidelberg.de/vigra/doc/vigra/documents/DataAccessors.ps">This paper</a> contains
     a detailed description of the concept. Here is a brief list of the basic
     accessor requirements:
 <p>
Index: b/include/vigra/functorexpression.hxx
===================================================================
--- a/include/vigra/functorexpression.hxx
+++ b/include/vigra/functorexpression.hxx
@@ -318,7 +318,7 @@
     \endcode
     
     Further information about how this mechanism works can be found in
-    <a href="documents/FunctorFactory.ps">this paper</a> (sorry, slightly out of date).
+    <a href="http://hci.iwr.uni-heidelberg.de/vigra/doc/vigra/documents/FunctorFactory.ps">this paper</a> (sorry, slightly out of date).
 */
 
 #ifndef DOXYGEN
Index: b/include/vigra/imageiterator.hxx
===================================================================
--- a/include/vigra/imageiterator.hxx
+++ b/include/vigra/imageiterator.hxx
@@ -554,7 +554,7 @@
     \endcode
 
     See the paper: U. Koethe:
-    <a href="documents/GenericProg2D.ps">Reusable Algorithms in Image Processing</a>
+    <a href="http://hci.iwr.uni-heidelberg.de/vigra/doc/vigra/documents/GenericProg2D.ps">Reusable Algorithms in Image Processing</a>
     for a discussion of the concepts behind ImageIterators.
 
 */
@@ -838,7 +838,7 @@
     Most functions and local types are inherited from ImageIteratorBase.
 
     See the paper: U. Koethe:
-    <a href="documents/GenericProg2D.ps">Reusable Algorithms in Image Processing</a>
+    <a href="http://hci.iwr.uni-heidelberg.de/vigra/doc/vigra/documents/GenericProg2D.ps">Reusable Algorithms in Image Processing</a>
     for a discussion of the concepts behind ImageIterators.
 
     <b>\#include</b> \<vigra/imageiterator.hxx\>
